src/dmd/dinterpret.d

```diff
@@ -88,31 +88,28 @@ public Expression ctfeInterpretForPragmaMsg(Expression e)
 
     // It's also OK for it to be a function declaration (happens only with
     // __traits(getOverloads))
-    if (e.op == TOK.variable && (cast(VarExp)e).var.isFuncDeclaration())
-    {
-        return e;
-    }
+    if (auto ve = e.isVarExp())
```

Comment:
I know it's not necessary but I think one should always but braces on an `if` statement if the body contains multiple lines.